src/bacnet/basic/service/h_cov.c

```diff
@@ -382,6 +382,7 @@ static bool cov_list_subscribe(BACNET_ADDRESS *src,
                     cov_address_remove_unused();
                 } else {
                     COV_Subscriptions[index].dest_index = cov_address_add(src);
+    fprintf(stderr, "[%s %d %s] dest_index = %d\r\n", __FILE__, __LINE__, __func__, COV_Subscriptions[index].dest_index);
```

Comment:
you might be mixing tabs and spaces in the code as some are misaligned. 
I see you removed the other stderr printouts, presumably you wanna keep these?